.github/workflows/intelligence-service-qa.yml

```
@@ -24,7 +24,7 @@ jobs:
       - name: Install Poetry
         uses: snok/install-poetry@v1
         with:
-          version: 1.8.5
+          version: 2.1.1
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)